types/api/v1beta1/event.proto

```diff
@@ -0,0 +1,126 @@
+syntax = "proto3";
+
+option go_package = "github.co/aquasecurity/tracee/api/v1beta1";
+
+package tracee.v1beta1;
+
+import "google/protobuf/timestamp.proto";
+
+
+message Event {
+    google.protobuf.Timestamp timestamp = 1;
+    uint32 id = 2;
+    string name = 3;
+    Policies policies = 4;
+    Context context = 5;
+    // repeated EventData data = 6; TODO: event data is coming on a separate PR
+    optional Threat threat = 7;
+}
+
+message Policies {
+    repeated string matched = 1;
+}
+
+message Context {
+    optional Process process = 1;
+    optional Container container = 2;
+    optional K8S k8s = 3;
+}
+
+message Process  {
+    Executable executable = 1;
+    int32 pid = 2;
+    uint32 entity_id = 3;
+    int32 namespaced_pid = 4;
```

Comment:
nit: better switch pid and entity_id places since pid better be near namespaced_pid